src/_zkapauthorizer/_storage_client.py

```diff
@@ -50,7 +50,7 @@
     int,
     tuple[
         list[
-            tuple[int, int, bytes, bytes],
+            tuple[int, int, bytes],
```

Comment:
This type used to be correct but the representation inside Tahoe-LAFS changed.  Our implementation was already updated (so it would work) but not the type.